### DISCUSSION TITLE
想要一个统计全部文章总字数的组件该怎么做

### DISCUSSION THREAD

**Discussion by Macxx-could** [halo-dev/halo#4607] [Q&A]
在仪表板这里希望能有一个统计全部文章总字数的组件，需要如何开发呢？
![image](https://github.com/halo-dev/halo/assets/82356512/4952f9f3-5412-4675-82f3-cd14e07442f7)

希望大佬们指条明路。

**Comment by ruibaby** [comment]
1. 通过插件获取所有文章的字数，并通过 API 暴露给前端。
2. Console 仪表盘的 Widgets 需要可扩展。
